### ISSUE TITLE
Open PR in Container: No prompt for dev container definition

### ISSUE THREAD

**Issue by aeschli** [microsoft/vscode-remote-release#2891]
Verifying https://github.com/microsoft/vscode-remote-release/issues/2771

- in the VSCode reop, open the PR extension
- select a PR and run `Checkout Pull Request in Dev Container`
- new window opens with error dialog
![image](https://user-images.githubusercontent.com/6461412/80600795-35dc1a80-8a2d-11ea-9e9a-6a615f660bf7.png)

According to https://github.com/microsoft/vscode-remote-release/issues/2771#issuecomment-618607536 there should be a dev container selection dialog

**Comment by aeschli** [comment]
- close remote
- recent menu has a funny entry
![image](https://user-images.githubusercontent.com/6461412/80600948-691ea980-8a2d-11ea-820e-b7214a061088.png)

**Comment by egamma** [comment]
closing, can no longer reproduce.

**Comment by aeschli** [comment]
When I try `Check out Pull Request in Dev Container` now in the vscode repo, nothing happens. The dev tools console gets an entry:
```
console.ts:137 [Extension Host] TypeError: i.start is not a function    at /home/martin/.vscode-insiders/extensions/ms-vscode-remote.remote-containers-0.165.0/dist/extension/extension.js:115:996    at z (/home/martin/.vscode-insiders/extensions/ms-vscode-remote.remote-containers-0.165.0/dist/extension/extension.js:114:505)    at ce (/home/martin/.vscode-insiders/extensions/ms-vscode-remote.remote-containers-0.165.0/dist/extension/extension.js:202:2458)    at Ep (/home/martin/.vscode-insiders/extensions/ms-vscode-remote.remote-containers-0.165.0/dist/extension/extension.js:270:3504)    at lr (/home/martin/.vscode-insiders/extensions/ms-vscode-remote.remote-containers-0.165.0/dist/extension/extension.js:270:1970)    at zq (/home/martin/.vscode-insiders/extensions/ms-vscode-remote.remote-containers-0.165.0/dist/extension/extension.js:271:36149)    at processTicksAndRejections (internal/process/task_queues.js:97:5) (at Ep (/home/martin/.vscode-insiders/extensions/ms-vscode-remote.remote-containers-0.165.0/dist/extension/extension.js:270:3610))
v @ console.ts:137
$logExtensionHostMessage @ mainThreadConsole.ts:35
_doInvokeHandler @ rpcProtocol.ts:417
_invokeHandler @ rpcProtocol.ts:402
_receiveRequest @ rpcProtocol.ts:318
_receiveOneMessage @ rpcProtocol.ts:245
(anonymous) @ rpcProtocol.ts:110
fire @ event.ts:622
fire @ ipc.net.ts:468
_receiveMessage @ ipc.net.ts:821
(anonymous) @ ipc.net.ts:660
fire @ event.ts:622
acceptChunk @ ipc.net.ts:241
(anonymous) @ ipc.net.ts:202
T @ ipc.net.ts:32
emit @ events.js:315
addChunk @ _stream_readable.js:295
readableAddChunk @ _stream_readable.js:271
Readable.push @ _stream_readable.js:212
onStreamRead @ internal/stream_base_commons.js:186
console.ts:137 [Extension Host] rejected promise not handled within 1 second: TypeError: t.output.write is not a function (at /usr/share/code-insiders/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:105:30306)
v @ console.ts:137
$logExtensionHostMessage @ mainThreadConsole.ts:35
_doInvokeHandler @ rpcProtocol.ts:417
_invokeHandler @ rpcProtocol.ts:402
_receiveRequest @ rpcProtocol.ts:318
_receiveOneMessage @ rpcProtocol.ts:245
(anonymous) @ rpcProtocol.ts:110
fire @ event.ts:622
fire @ ipc.net.ts:468
_receiveMessage @ ipc.net.ts:821
(anonymous) @ ipc.net.ts:660
fire @ event.ts:622
acceptChunk @ ipc.net.ts:241
(anonymous) @ ipc.net.ts:202
T @ ipc.net.ts:32
emit @ events.js:315
addChunk @ _stream_readable.js:295
readableAddChunk @ _stream_readable.js:271
Readable.push @ _stream_readable.js:212
onStreamRead @ internal/stream_base_commons.js:186
console.ts:137 [Extension Host] stack trace: TypeError: t.output.write is not a function    at Ep (/home/martin/.vscode-insiders/extensions/ms-vscode-remote.remote-containers-0.165.0/dist/extension/extension.js:270:3685)    at processTicksAndRejections (internal/process/task_queues.js:97:5)    at lr (/home/martin/.vscode-insiders/extensions/ms-vscode-remote.remote-containers-0.165.0/dist/extension/extension.js:270:1964)    at zq (/home/martin/.vscode-insiders/extensions/ms-vscode-remote.remote-containers-0.165.0/dist/extension/extension.js:271:36143) (at /usr/share/code-insiders/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:105:30385)
v @ console.ts:137
$logExtensionHostMessage @ mainThreadConsole.ts:35
_doInvokeHandler @ rpcProtocol.ts:417
_invokeHandler @ rpcProtocol.ts:402
_receiveRequest @ rpcProtocol.ts:318
_receiveOneMessage @ rpcProtocol.ts:245
(anonymous) @ rpcProtocol.ts:110
fire @ event.ts:622
fire @ ipc.net.ts:468
_receiveMessage @ ipc.net.ts:821
(anonymous) @ ipc.net.ts:660
fire @ event.ts:622
acceptChunk @ ipc.net.ts:241
(anonymous) @ ipc.net.ts:202
T @ ipc.net.ts:32
emit @ events.js:315
addChunk @ _stream_readable.js:295
readableAddChunk @ _stream_readable.js:271
Readable.push @ _stream_readable.js:212
onStreamRead @ internal/stream_base_commons.js:186
log.ts:296   ERR t.output.write is not a function: TypeError: t.output.write is not a function
    at Ep (/home/martin/.vscode-insiders/extensions/ms-vscode-remote.remote-containers-0.165.0/dist/extension/extension.js:270:3685)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at lr (/home/martin/.vscode-insiders/extensions/ms-vscode-remote.remote-containers-0.165.0/dist/extension/extension.js:270:1964)
    at zq (/home/martin/.vscode-insiders/extensions/ms-vscode-remote.remote-containers-0.165.0/dist/extension/extension.js:271:36143)
```

**Comment by aeschli** [comment]
Let me know if you want a new issue for that.

**Comment by egamma** [comment]
@aeschli this is https://github.com/microsoft/vscode-remote-release/issues/4754 which got fixed in 0.165.1.

**Comment by rzhao271** [comment]
Marking as verified because I see the prompt again. 
I am assuming https://github.com/microsoft/vscode-remote-release/issues/2891#issuecomment-621202697 is by design.
